tests/snappi_tests/multidut/bgp/files/bgp_outbound_helper.py

```diff
@@ -0,0 +1,632 @@
+        "-c 'set ipv6 next-hop prefer-global' "
+        "-c 'exit' "
+        "-c 'ip nht resolve-via-default' "
+        "-c 'ipv6 nht resolve-via-default' "
+        "-c 'ipv6 protocol bgp route-map RM_SET_SRC6' "
+        "-c 'router bgp %s' "
+        "-c 'neighbor %s remote-as %s' "
+        "-c 'address-family ipv4 unicast' "
+        "-c 'neighbor %s activate' "
+        "-c 'neighbor %s remote-as %s' "
+        "-c 'address-family ipv6 unicast' "
+        "-c 'neighbor %s activate' "
+        "-c 'neighbor %s soft-reconfiguration inbound' "
+        "-c 'neighbor %s route-map RM_SET_SRC6 in' "
+        "-c 'maximum-paths 64' "
+        "-c 'exit' "
+    )
+    t2_downlink_bgp %= (
+        t2_side_interconnected_port['asic_value'][-1], T2_DUT_AS_NUM,
+        t1_t2_dut_ipv4_list[index], T1_DUT_AS_NUM, t1_t2_dut_ipv4_list[index],
+        t1_t2_dut_ipv6_list[index], T1_DUT_AS_NUM, t1_t2_dut_ipv6_list[index],
+        t1_t2_dut_ipv6_list[index], t1_t2_dut_ipv6_list[index]
+    )
+    duthosts[2].shell(t2_downlink_bgp)
+    logger.info('Configuring BGPv4 and BGP+ Neighbor {} {} in {} frr {}'.format(t1_t2_dut_ipv4_list[index],
+                t1_t2_dut_ipv6_list[index], duthosts[2].hostname, t2_side_interconnected_port['asic_value'][-1]))
+
+    logger.info('--------------- T2 Uplink - Tgen Section --------------------')
+
+    index = 0
+    for asic_value, portchannel_info in t2_uplink_portchannel_members[duthosts[1].hostname].items():
+        for portchannel, ports in portchannel_info.items():
+            duthosts[1].command('sudo config portchannel -n {} add {} \n'.
+                                format(asic_value, portchannel))
+            logger.info('\n')
+            logger.info('Adding -n {} {} in {}'.format(asic_value, portchannel, duthosts[1].hostname))
+            for port in ports:
+                duthosts[1].command('sudo config portchannel -n {} member add {} {} \n'.
+                                    format(asic_value, portchannel, port))
+                logger.info('Adding member {} to {}'.format(port, portchannel))
+
+            logger.info('Configuring IPs {} /  {}  to {}'.
+                        format(t2_dut_portchannel_ipv4_list[index], t2_dut_portchannel_ipv6_list[index], portchannel))
+            duthosts[1].command('sudo config interface -n {} ip add {} {}/{} \n'.
+                                format(asic_value, portchannel, t2_dut_portchannel_ipv4_list[index], v4_prefix_length))
+            duthosts[1].command('sudo config interface -n {} ip add {} {}/{} \n'.
+                                format(asic_value, portchannel, t2_dut_portchannel_ipv6_list[index], v6_prefix_length))
+            index = index + 1
+
+    index = 0
+    for asic_value, portchannel_info in t2_uplink_portchannel_members[duthosts[1].hostname].items():
+        for portchannel, ports in portchannel_info.items():
+            logger.info('\n')
+            bgp_config = (
+                "vtysh -n %s "
```

Comment:
handle single asic case here as well.

Reply:
Currently it is written by assuming it's a multi-asic linecard. We will incorporate the changes to support single asic linecard also.

Reply:
fixed